apps/nerves_hub_www/lib/nerves_hub_www_web/views/org_user_view.ex

```diff
@@ -1,3 +1,13 @@
 defmodule NervesHubWWWWeb.OrgUserView do
   use NervesHubWWWWeb, :view
+
+  defmodule DateTimeFormat do
+    @months %{1 => "Jan", 2 => "Feb", 3 => "Mar", 4 => "Apr",
+      5 => "May", 6 => "Jun", 7 => "Jul", 8 => "Aug",
+      9 => "Sep", 10 => "Oct", 11 => "Nov", 12 => "Dec"}
+    
+    def format(timestamp) do
+      '#{@months[timestamp.month]} #{timestamp.day}, #{timestamp.year}'
+    end
+  end
```

Comment:
Would it make sense to move this to a location where it would be more readily accessible by other views templates or will we only be needing it here?

Reply:
I could definitely see it being used elsewhere. Where do you think an appropriate place for it would be?

Reply:
Maybe the LayoutView? Are those functions made available throughout the views?

Reply:
Not by default no, we'd still have to import it but that's probably the most generic place at the moment